04-ruby-mvc-sinatra/active-record-intro/readme.md

```diff
@@ -492,18 +500,30 @@ adam = Instructor.create(first_name: "adam", last_name: "bray", age: 30)
 robin.students.create(first_name: "michael", last_name: "scott", age: 45, job: "office manager")
 robin.students.create(first_name: "Dwight", last_name: "Schrute", age: 34, job: "Assistant to the Regional Manager")
 adam.students.create(first_name: "Dee", last_name: "Reynolds", age: 32, job: "Bartender")
-adam.students.create(first_name: "George", last_name: "Costanza", age: 45, job: "toner seller")
+adam.students.create(first_name: "Charlie", last_name: "Kelly", age: 31, job: "Owner of Paddy's")
 
 ```
 
 ### Seeding a database - Hopsital ( you do- 10m)
-- create a seed file that contains all dependencies and establish connection to database /w Active Record
-- in the seed file create at least 2 doctors and 4 patients
-- make sure you destroy all objects before creating new ones
+1. Create a seed file that contains all dependencies and establish connection to database /w Active Record
+2. In the seed file, create at least 2 doctors and 4 patients
+3. Make sure you destroy all objects before creating new ones
+
+## Closing
+Who misses writing SQL queries by hand? Exactly. Active Record is extremely powerful and helpful to more easily interface with the business models for our applications.
```

Comment:
fyi first time i tought active record I got a lot of push back - like why the eff did you make us learn sql if we're not gonna use it anymore? consider that when preparing framing